packages/backend/src/engine/backend-secret/backend_secret_fetcher_factory.ts

```diff
@@ -33,15 +52,18 @@ export class BackendSecretFetcherFactory {
     scope: Construct,
     secretName: string,
     backendIdentifier: BackendIdentifier
-  ): CustomResource => {
-    const secretResourceId = `${secretName}SecretFetcherResource`;
+  ): SecretFetcherCustomResource => {
+    const secretResourceId = `SecretFetcherResource`;
```

Comment:
```suggestion
    const secretResourceId = `AmplifySecretFetcherResource`;
```

just in case.

Reply:
updated (AmplifySecretFetcherCustomResource)

Reply:
I meant resource id not the class name.